src/components/ttt-tile/ttt-tile.scss

```diff
@@ -0,0 +1,113 @@
+@import "../../colors";
+@import "../../frameless";
+
+.ttt-tile {
+    display: inline-block;
+    // work-around chrome bug with columns support - 1 px of next column ends up
+    // at the bottom of the previous column.
+    margin-top: 1px;
+    margin-bottom: calc(1.25rem - 1px);
+
+    border-radius: 1rem;
+    box-shadow: 0 0 0 1px $active-gray;
```

Comment:
Nope, Carl had sent me rgb values and this was probably just the first one I saw that matched - I'll make the change.